src/course/config.json

```diff
@@ -5,7 +5,8 @@
     "_accessibility": {
         "_isEnabled": false,
         "_shouldSupportLegacyBrowsers": true,
-        "_isTextProcessorEnabled": true
+        "_isTextProcessorEnabled": true,
+        "_disableTextSelectOnClasses": ".size-small .size-medium .size-large"
```

Comment:
This bit would become:

"_disableTextSelectOnClasses": ".touch.size-small"

for mobile touch devices